.github/workflows/ci.yml

```diff
@@ -71,4 +71,21 @@ jobs:
                 curl -L -O -k https://gitlab.tiker.net/inducer/ci-support/raw/master/build-and-test-py-project.sh
                 . ./build-and-test-py-project.sh
 
+    docs:
+        name: Documentation
+        runs-on: ubuntu-latest
+        steps:
+        -   uses: actions/checkout@v2
+        -
+            uses: actions/setup-python@v1
+            with:
+                python-version: '3.x'
+        -   name: "Main Script"
+            run: |
+                EXTRA_INSTALL="numpy"
+                curl -L -O -k https://gitlab.tiker.net/inducer/ci-support/raw/master/ci-support.sh
```

Comment:
Can this be replaced by:
```
curl -L -O -k https://gitlab.tiker.net/inducer/ci-support/raw/master/build-docs.sh
. build-docs.sh
```
?

Reply:
I'd prefer not. Sourcing `ci-support.sh` and then calling functions from it is now the preferred way.